[BUG] Incorrectly listing new documents as duplicates

### Description

I use dockerized version 1.9.2  (image: ghcr.io/paperless-ngx/paperless-ngx:latest)
and use usually use a consume folder for feeding paperless. I just recently updated to that version.
I have now several documents that are kept in that folder and are shown in the log as: "Not consuming" as "it is a duplicate". 

However that is not correct, there are no duplicates to be found. From the task view I have only the option to abort, but no option to overwrite and ignore. I also seem to have no option to delete the "false original".

### Steps to reproduce

add document into consume folder.
Receive error

### Webserver logs

```bash
[2022-09-30 10:29:23,621] [ERROR] [paperless.consumer] Not consuming 1802140_Stromabrechnung_2020.pdf: It is a duplicate.
```


### Paperless-ngx version

1.9.2

### Host OS

Synology Docker

### Installation method

Other (please describe above)

### Browser

_No response_

### Configuration changes

_No response_

### Other

_No response_

--- a/src/documents/consumer.py
+++ b/src/documents/consumer.py
@@ -120,7 +120,7 @@ class Consumer(LoggingMixin):
             self._fail(
                 MESSAGE_DOCUMENT_ALREADY_EXISTS,
                 f"Not consuming {self.filename}: It is a duplicate of"
-                f" {existing_doc.get().title}",
+                f" {existing_doc.get().title} (#{existing_doc.get().pk})",
             )
 
     def pre_check_directories(self):

--- a/src/documents/consumer.py
+++ b/src/documents/consumer.py
@@ -111,14 +111,16 @@ class Consumer(LoggingMixin):
     def pre_check_duplicate(self):
         with open(self.path, "rb") as f:
             checksum = hashlib.md5(f.read()).hexdigest()
-        if Document.objects.filter(
+        existing_doc = Document.objects.filter(
             Q(checksum=checksum) | Q(archive_checksum=checksum),
-        ).exists():
+        )
+        if existing_doc.exists():
             if settings.CONSUMER_DELETE_DUPLICATES:
                 os.unlink(self.path)
             self._fail(
                 MESSAGE_DOCUMENT_ALREADY_EXISTS,
-                f"Not consuming {self.filename}: It is a duplicate.",
+                f"Not consuming {self.filename}: It is a duplicate of"
+                f" {existing_doc.get().title}",
             )
 
     def pre_check_directories(self):

